webapp/src/components/surveyPost/surveyPost.tsx

```diff
@@ -224,7 +230,17 @@ function SurveyPost({post}: CustomPostTypeComponentProps) {
                 {
                     disabled && surveyExpired &&
                     <div className='surveyMessage submitted'>
-                        {`Survey expired on ${surveyExpireAtDate?.toLocaleDateString()}.`}
+                        {
+                            surveyExpireAtDate &&
```

Comment:
0/5 but it might be worth turning these two cases into a ternary so that it's clear that they're handling opposite cases. I am a bit leery about the amount of conditional logic in this JSX, but I do really like how you've laid it out with a lot of whitespace to keep it more readable

Reply:
ternary operator makes so much sense. Done!